src/renderer/components/item-object-list/item-list-layout.tsx

```diff
@@ -322,7 +323,7 @@ export class ItemListLayout<I extends ItemObject> extends React.Component<ItemLi
 
   renderNoItems() {
     if (this.failedToLoad) {
-      return <NoItems>Failed to load items.</NoItems>;
+      return <NoItems>Failed to load items.{this.props.renderFailedToLoadDetails?.()}</NoItems>;
```

Comment:
```suggestion
      if(this.props.renderFailedToLoadDetails) {
       return <NoItems>{this.props.renderFailedToLoadDetails()}</NoItems>
      }

      return <NoItems>Failed to load items.</NoItems>;
```

Otherwise I would simplify to `{noItemsDetails?: React.ReactNode}` for partial customization.

Reply:
Good idea, will change to react node